src/touchstone/OCM-benchmark test(1).py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
lets create a new repository for juypter, lets not have it in benchmark-comparison.

Reply:
Okay, got it